.travis.yml

```diff
@@ -48,6 +48,23 @@ jobs:
       script:
         - make cephfsplugin
 
+after_script:
+
+# Make root mounted as rshared to fix kube-dns issues.
+- sudo mount --make-rshared /
+# Download kubectl, which is a requirement for using minikube.
+- curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.13.0/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+# Download minikube.
+- curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
```

Comment:
I would stick to a minikube version than use latest each time, so that the CI environment is a constant, than a changing environment as we pick latest of another dependency for each PR or merge.

As a rule I would like to see us using fixed versions than picking up latest of any dependency if possible.

Reply:
Done.